### DISCUSSION TITLE
魈宝世界第一可爱可爱

### DISCUSSION THREAD

**Discussion by 3469692857** [share121/inter-knot#2964] [其他]
魈厨日常犯病
![IMG_20240803_213313](https://github.com/user-attachments/assets/a4f43558-c824-4c8b-8f3b-29da1cd33210)
![202433013850502](https://github.com/user-attachments/assets/ddab78ee-f23e-4c70-9f28-8346a13ac6bb)

**Comment by chenx198** [comment]
流萤 欸嘿 我的流萤 ![屏幕截图 2024-06-22 160419](https://github.com/user-attachments/assets/14cf1a7d-ea39-4d7c-bac4-c03f367e6b23)

**Comment by Joker66616** [comment]
嘿嘿嘿风系小男孩什么的最好了<img src="https://img-static.mihoyo.com/communityweb/upload/e52a3fa8fd52d7577293332d8c704d67.png" alt="且听风吟" width="75" >

**Reply by 3469692857** [reply]
同意ớ ₃ờ
